app/src/main/java/com/vendetta/xposed/Main.kt

```diff
@@ -95,7 +116,51 @@ class Main : IXposedHookZygoteInit, IXposedHookLoadPackage {
             }
         }
 
+        @Suppress("UNCHECKED_CAST")
+        val drawableIdPatch = object: XC_MethodHook() {
```

Comment:
These patches should just be defined anonymously in the `hookMethod` call. The only reason `path` is separated out is because I patch two methods with the same patch.

Reply:
i just felt it made things more organized